test/helpers/describe_installer.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const _ = require('lodash')
+const crypto = require('crypto')
+const fs = require('fs-extra')
+const os = require('os')
+const path = require('path')
+
+const installer = require('../..')
+
+module.exports = function describeInstaller (description, installerOptions, itDescription, itFunc) {
+  describe(description, test => {
+    const outputDir = module.exports.tempOutputDir()
```

Comment:
could we also add the option to set a `dest` directory coming from the `installerOptions`? It's mainly to make it easier for debugging when I need to specify a particular folder for a specific test. It would look something like this, assuming we're using the `temp` package:
`const outputDir = installerOptions.dest || temp.path('electron-installer-debian-')`

Reply:
I don't see why not.